samples/cpp/build_samples.sh

```diff
@@ -88,22 +91,21 @@ OS_PATH=$(uname -m)
 NUM_THREADS="-j2"
 
 if [ "$OS_PATH" == "x86_64" ]; then
-  OS_PATH="intel64"
-  NUM_THREADS="-j8"
+    OS_PATH="intel64"
+    NUM_THREADS="-j8"
 fi
 
-if [ -e "$build_dir/CMakeCache.txt" ]; then
-  rm -rf "$build_dir/CMakeCache.txt"
+if [ -e "$samples_build_dir/CMakeCache.txt" ]; then
+    rm -rf "$samples_build_dir/CMakeCache.txt"
 fi
 
-mkdir -p "$build_dir"
-cd "$build_dir" || exit
-$CMAKE_EXEC -DCMAKE_BUILD_TYPE=Release "$SAMPLES_PATH"
-make $NUM_THREADS
+mkdir -p "$samples_build_dir" && cd "$samples_build_dir"
```

Comment:
```suggestion
mkdir -p "$samples_build_dir"
cd "$samples_build_dir"
```
You have `set -e`. By the way, I remember, that `set -e` was there. Seems that someone deleted this :)

Reply:
it's just to shorten the boilerplate code

Reply:
The point is that now you don't need && between mkdir and cd

Reply:
reverted